pypeit/edgetrace.py

```diff
@@ -2542,7 +2544,7 @@ def check_synced(self, rebuild_pca=False):
                 msgs.info('Found {0} slit(s) with gaps below {1} arcsec ({2:.2f} pixels).'.format(
                             np.sum(indx), self.par['minimum_slit_gap'], gap_atol))
                 rmtrace = np.concatenate(([False],np.repeat(indx,2),[False]))
-                self.remove_traces(rmtrace, rebuild_pca=rebuild_pca)
+                self.remove_traces(rmtrace, rebuild_pca=_rebuild_pca)
```

Comment:
I take this occasion to mention that I found a typo in line 2514. The variable defined there should be `dlength_atol` and not `length_atol`, which is defined 2 lines below that.

Reply:
Thanks for finding that!! Definitely a bug.  Fixed.